docs/sources/administration/manage-users-and-permissions/manage-server-users/remove-server-user-from-org.md

```diff
@@ -0,0 +1,21 @@
++++
+title = "Remove a user from an organization"
+aliases = ["docs/sources/administration/manage-users-and-permissions/manage-server-users/remove-server-user-from-org.md"]
+weight = 40
++++
+
+# Remove a user from an organization
```

Comment:
I think this section should appear on the same page as Add User to Organization, since it's confusing to navigate between pages to perform similar actions.

Reply:
Done.